tests/benchmarks/micro_benchmarks/test_gpu_burn_test.py

```diff
@@ -57,4 +57,7 @@ def test_gpu_burn(self, results):
         assert (benchmark.result['time'][0] == time)
         for device in range(8):
```

Comment:
Should we add some correctness check?

Reply:
updated correctness check according to current static test data file